### PR TITLE
distilgpt2-hftransformersv2

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
distilgpt2-hftransformersv2

### CODE REVIEW

**Review by novaturient95 — APPROVED** [review]

**Comment by KonakanchiSwathi** [comment]
job link:- failed in token classifiacation compute metrics  but proceeded with pr request 
https://ml.azure.com/experiments/id/0f020411-bbcd-4256-adcb-b55ba303a32a?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

**Comment by novaturient95** [comment]
Same as gpt2 PR. Please either include all models changes in one PR and close remaining PRs or segregate to keep only the changes specific to a model mentioned in PR title

File: assets/models/system/bert-base-cased/description.md [hunk 1]
```diff
@@ -31,12 +31,14 @@ Fill Mask|Fill Mask|<a href="https://huggingface.co/datasets/rcds/wikipedia-for-
 #### Sample input
 ```json
 {
-    "inputs": {
```
Comment (novaturient95) [inline assets/models/system/bert-base-cased/description.md:34]: Same as gpt2 PR. Please either include all models changes in one PR and close remaining PRs or segregate to keep only the changes specific to a model mentioned in PR title

**Review by novaturient95 — CHANGES_REQUESTED** [review]

**Comment by KonakanchiSwathi** [comment]
closing this pr request
https://github.com/Azure/azureml-assets/pull/1181

**Pull request closed without merging**